lib/telemetry_metrics_statsd.ex

```diff
@@ -207,6 +207,7 @@ defmodule TelemetryMetricsStatsd do
           | {:mtu, non_neg_integer()}
           | {:prefix, prefix()}
           | {:formatter, :standard | :datadog}
+          | {:default_tags, Keyword.t()}
```

Comment:
What do you think about `global_tags` instead of `default_tags`?

Reply:
I don't necessarily _love_ either but I'm fine with either. It feels like the intent is "a set of tags you will get for free unless you explicitly override one or more of them". Neither "global" nor "default" seem to communicate that as cleanly as I might hope. But I can't think of a nicer word for it so I'm fine with `global_tags`.

Reply:
Hm, I agree. Although "global" sounds slightly better to me than "default", none is ideal. What about "static"? I'm not sure if that communicates the intent either.

Reply:
Yeah :/ It's not necessarily static either though since you can override. "global" seems fine for now.